pkger/parser_models.go

```diff
@@ -657,6 +659,7 @@ func (c chart) properties() influxdb.ViewProperties {
 			XColumn:           c.XCol,
 			YColumn:           c.YCol,
 			ShadeBelow:        c.Shade,
+			HoverDimension:    c.HoverDimension,
```

Comment:
to confirm, the hover dimension is only valid for single stat plus line and the xy views?

Reply:
Correct